app/src/main/java/net/osmtracker/db/DatabaseHelper.java

```diff
@@ -125,7 +126,7 @@
 	 * v17: add TBL_TRACKPOINT.COL_ATMOSPHERIC_PRESSURE and TBL_WAYPOINT.COL_ATMOSPHERIC_PRESSURE
 	 *</pre>

```

Comment:
Update documentation:

` 	 * v18: add TBL_TRACKPOINT.COL_NEW_SEGMENT`

Reply:
Thanks for your feedback

> would be nicer that every point get associated with a consecutive segment number and not with 0 or 1 only. What do you think?

You mean, every segment? Yes, that would be another way to represent multiple segments.
The way the my proposal currently works is that the *first* point of each segment has 1 (or true), and all the following ones are false again. In a way, it is the derivative of the segment number.
However, if you have a strong preference for the other way, I can change it. However, in that case, I'll apply this to my later pull request (#312, "Import a gpx file as a route to follow"), because there is some interdependence among both.  Also, please note that I will be on the move until end of August, and might not get back to this until then.